src/Designer/frontend/libs/studio-browser-storage/package.json

```
@@ -0,0 +1,18 @@
+{
+  "name": "@studio/browser-storage",
+  "packageManager": "yarn@4.9.4",
+  "version": "0.1.0",
+  "main": "./src/index.ts",
+  "scripts": {
+    "test": "jest",
+    "lint": "eslint \"src/**/*.ts\""
```

Savner `lint:fix` og `typecheck` her
```suggestion
    "test": "jest",
    "lint": "eslint \"src/**/*.ts\"",
    "lint:fix": "yarn run lint --fix",
    "typecheck": "tsc --noEmit"
```

---

Bra catch!